File: JavaScript/1144.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split('\n');

let entrada = parseInt(lines[0])

let b, c, d, e, f;

for(b = 1; b <= entrada; b++){
    c = b * b;

    d = b * b * b;

    console.log(`${b} ${c} ${d}`)

    e = c + 1;

    f = d + 1;

    console.log(`${b} ${e} ${f}`)
}

File: C++/1144.cpp
#include <iostream>

using namespace std;

int main(){
    int entrada, b, c, d, e, f;

    cin >> entrada;

    for(b = 1; b <= entrada; b++){
        c = b * b;
        d = b * b * b;

        cout << b << " " << c << " " << d << endl;

        e = c + 1;
        f = d + 1;

        cout << b << " " << e << " " << f << endl;
    }

    return 0;
}